### ISSUE TITLE
RNN with GDN Paper in CVPR 2021

### ISSUE THREAD

**Issue by khawar-islam** [1zb/pytorch-image-comp-rnn#19]
Dear @1zb ,
Greetings!

**New Version of RNN with GDN**
[Paper] "Islam, Khawar, L. Minh Dang, Sujin Lee, and Hyeonjoon Moon. "Image Compression With Recurrent Neural Network and Generalized Divisive Normalization." In Proceedings of the IEEE/CVF Conference on Computer Vision and Pattern Recognition, pp. 1875-1879. 2021."
[Code] https://khawar512.github.io/cvpr/
